index.html

```diff
@@ -180,49 +176,49 @@ <h2 hidden>Наши партнеры</h2>
 
   </main>
 
-  <footer class="main_footer">
+  <footer class="main-footer">
 
-    <section class="footer_location">
+    <section class="footer-location">
       <h2 hidden>Как нас найти</h2>
       <h3>NЁRDS Design Studio</h3>
       <p>наб. Реки Карповки, 5 лит П, Санкт-Петербург</p>
       <p>
-        <a class="footer_phone" href="tel: +7 800 555-86-28">+7 800 555-86-28</a>
+        <a class="footer-phone" href="tel: +7 800 555-86-28">+7 800 555-86-28</a>
       </p>
       <p>
-        <a class="button red_button" href="#">Напишите нам</a>
+        <a class="button red-button" href="#">Напишите нам</a>
       </p>
       <p><img src="img/map.png" width="1440" height="416" alt="Карта"></p>
       <!--<img (здесь будет маркер) width="231" height="190" alt="Маркер на карте">-->
     </section>
 
-    <section class="footer_contacts list">
+    <section class="footer-contacts list">
       <h2 hidden>Как с нами связаться</h2>
 
       <ul class="list">
         <li>
-          <a class="social_button button" href="#">
+          <a class="social-button button" href="#">
             <img src="img/vk-icon.svg" width="26" height="15" alt="Вконтакте">
           </a>
         </li>
         <li>
-          <a class="social_button button" href="#">
+          <a class="social-button button" href="#">
             <img src="img/fb-icon.svg" width="12" height="22" alt="Фейсбук">
           </a>
         </li>
         <li>
-          <a class="social_button button" href="#">
+          <a class="social-button button" href="#">
             <img src="img/insta-icon.svg" width="21" height="21" alt="Инстаграм">
           </a>
         </li>
       </ul>
 
-      <p class="friendship fri_one">Давайте дружить, это выгодно!</p>
-      <p class="friendship fri_two">Скидка 10% для друзей из социальных сетей.</p>
+      <p class="friendship advantage">Давайте дружить, это выгодно!</p>
+      <p class="friendship social">Скидка 10% для друзей из социальных сетей.</p>
```

Comment:
Опять же, зачем тебе 2 одинаковых класса? Они совсем не похожи внешне.
И лучше искать суть элемента, в плане того, в какой роли он выступает в дизайне. Например, тут у нас внешне заголовок и текст.

Не заморачиваемся. Например, так:
```suggestion
      <p class="footer-title">Давайте дружить, это выгодно!</p>
      <p class="footer-text">Скидка 10% для друзей из социальных сетей.</p>
```